### DISCUSSION TITLE
[FEATURE]像PaaS一样，创建网站后可通过特定的Git仓库拉代码自动部署

### DISCUSSION THREAD

**Discussion by geekgao** [1Panel-dev/1Panel#6236] [Ideas]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

就像大部分PaaS一样，监听Git仓库，自动部署代码。
这样，结合已有的数据库和docker管理功能，基本上就实现了通用PaaS的基础功能了。
我觉得，这样会让你们产品提升一大档次，明显与BT、CyberPanel等面板拉开竞争水平线。

### 请描述你建议的实现方案

我个人感觉，可以只做个最基础的功能：从Git自动拉代码，然后根据一个基础配置脚本进行部署+运行。从环境变量中读取DB等配置信息，就像vercel。
我看了下/backend/app/service/website.go 代码，加入这部分代码应该不复杂。但是不知道是不是与现有产品定位有冲突。

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。由于目前资源有限，我们在中短期内无法支持 CI/CD 相关功能。我们会根据需求优先级进行调整，感谢理解与支持。
